internal/bundle/bundle.go

```diff
@@ -67,7 +68,7 @@ func Validate(ctx context.Context, imagePath string) (*Report, error) {
 			return nil, fmt.Errorf("%v", err)
 		}
 		if k8sVer, found := ocpToKubeVersion[targetVersion]; found {
-			log.L().Debugf("OpenShift %s detected in annotations. Running with additional checks enabled.", targetVersion)
+			logger.V(log.DEBUGLEVEL).Info("running with additional checks enabled", "version", targetVersion)
```

Comment:
Did we mean to remove the context related to the OpenShift version, and how that triggered additional checks?

Reply:
Something in a Debug statement triggered additional checks?

Reply:
The original log message said contained `OpenShift %s detected in annotations` but it doesn't exist in the newly formatted log statement message.

```suggestion
			logger.V(log.DEBUGLEVEL).Info("OpenShift %s detected in annotations. Running with additional checks enabled", "version", targetVersion)
```

This suggestion doesn't take into account whether this is a valid way to do string interpolation for this `Info` method.

Reply:
Oh... It's not string interpolation. It's structured logging. Instead of interpolation, you can use "msg, key, value, key, value, ..."

If you want to keep interpolation, then you would use fmt.Sprintf()